src/QueryBuilder/DSL/Aggregation.php

```diff
@@ -354,9 +354,9 @@ public function ipv4_range(string $name, string $field): IpRange
      *
      * @see https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-histogram-aggregation.html
      *
-     * @param string $name     the name of this aggregation
-     * @param string $field    the name of the field on which to perform the aggregation
-     * @param int    $interval the interval by which documents will be bucketed
+     * @param string     $name     the name of this aggregation
+     * @param string     $field    the name of the field on which to perform the aggregation
+     * @param int|string $interval the interval by which documents will be bucketed
      */
     public function histogram(string $name, string $field, $interval): Histogram
```

Comment:
```suggestion
    public function histogram(string $name, string $field, $interval = null): Histogram
```

To reflect the new constructor default value for `$interval`?
Should probably also be applied to `date_histogram` method in this class?

Reply:
fixed!